app/models/mixins/assignment_mixin.rb

```diff
@@ -77,6 +77,38 @@ def assign_to_labels(objects, klass)
     reload
   end
 
+  def unassign_objects(objects)
+    current_assignments = get_assigned_tos
+
+    remove_all_assigned_tos
+
+    current_assignments[:objects] = current_assignments[:objects] - objects
+    reassign(current_assignments)
+
+    get_assigned_tos
+  end
+
+  def unassign_tags(tags)
+    current_assignments = get_assigned_tos
+
+    remove_all_assigned_tos
```

Comment:
@elad661 I see remove_all_assigned_tos gets a category suffix as argument. 
Can we pass only the tags we want to remove?

Reply:
@moolitayer I tried that, it doesn't work if you pass a specific tag.